projects/shape_bias/toy_example.py

```diff
@@ -0,0 +1,154 @@
+dataset_type = 'ImageNet'
+data_root = 'data/imagenet/'
+data_preprocessor = dict(
+    num_classes=1000,
+    mean=[123.675, 116.28, 103.53],
+    std=[58.395, 57.12, 57.375],
+    to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile', file_client_args=dict(backend='disk')),
```

Comment:
```suggestion
    dict(type='LoadImageFromFile'),
```

file_client_args will be deprecated